core/arch/arm/plat-imx/config/imx6ul.h

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2018 NXP
+ *
+ */
+
+#ifndef PLATFORM_CONFIG_IMX6UL_H
+#define PLATFORM_CONFIG_IMX6UL_H
+
+#ifdef CFG_WITH_PAGER
+#error "Pager not supported for platform mx6ulevk"
+#endif
+#ifdef CFG_WITH_LPAE
+#error "LPAE not supported for now"
```

Comment:
6UL does not support LPAE. Please refine.

Reply:
ok